src/views/Orders.vue

```diff
@@ -1,20 +1,249 @@
+      </ion-toolbar>
+
+     
+      <ion-toolbar>
+        <ion-searchbar :placeholder="$t('Search Orders')"></ion-searchbar>
+      </ion-toolbar>
+
+      <ion-toolbar>
+        <ion-segment @ionChange="segmentChanged($event)" v-model="segmentName" >
+          <ion-segment-button value="open">
+            <ion-label>{{ $t("OPEN") }}</ion-label>
+          </ion-segment-button>
+          <ion-segment-button value="packed">
+            <ion-label>{{ $t("PACKED") }}</ion-label>
+          </ion-segment-button>
+        </ion-segment>
       </ion-toolbar>
     </ion-header>
-    
-    <ion-content :fullscreen="true">
-   
+
+
+    <ion-content>
+      <div v-if="segmentName == 'open'">
+        <ion-card>
+         <ion-card-header>
+            <ion-item lines="none">
+              <ion-label>
+                <h2>Customer Name</h2>
+                <p>Customer Id</p>
+              </ion-label>
+              <ion-note slot="end">order time delta</ion-note>
+            </ion-item>
+          </ion-card-header>
+
+          <ion-card-content>
+             <ion-list>
+              <ion-item lines="none">
+                <ion-thumbnail slot="start">
+                  <img src="https://images.all-free-download.com/images/graphicthumb/fashion_model_portrait_205201.jpg"/>
+                </ion-thumbnail>
+                <ion-label>
+                  <h5>BRAND</h5>
+                  <h2>Virtual Name</h2>
+                  <p>{{ $t("Color") }} : color</p>
+                  <p>{{ $t("Size") }} : size</p>
+                </ion-label>
+                <ion-note color="secondary">15 {{ $t("In Stock") }}</ion-note>
+              </ion-item>
+              <ion-item lines="full">
+                <ion-thumbnail slot="start">
+                  <img src="https://images.all-free-download.com/images/graphicthumb/fashion_model_portrait_205201.jpg" />
+                </ion-thumbnail>
+                <ion-label>
+                  <h5>BRAND</h5>
+                  <h2>Virtual Name</h2>
+                  <p>{{ $t("Color") }} : color</p>
+                  <p>{{ $t("Size") }} : size</p>
+                </ion-label>
+                <ion-note color="secondary">15 {{ $t("In Stock") }}</ion-note>
+              </ion-item>
+            </ion-list>
+          </ion-card-content>
+
+          <ion-item>
+            <ion-icon :icon="callOutline" slot="start" />
+            <ion-label>{{ $t("Phone Number") }}</ion-label>
+            <ion-button fill="outline" slot="end" color="medium">
+              {{ $t("Copy") }}
+            </ion-button>
+          </ion-item>
+
+          <ion-item lines="full">
+            <ion-icon :icon="mailOutline" slot="start" />
+            <ion-label>{{ $t("Email") }}</ion-label>
+            <ion-button fill="outline" slot="end" color="medium">
+              {{ $t("Copy") }}
+            </ion-button>
+          </ion-item>
+
+          <div>
+            <ion-button fill="clear" slot="start">
```

Comment:
we shouldn't use a button slot to place the label of a button. Instead the button will align to the start automatically if it is placed as a block element in the flow of the template.

Reply:
This is done. Thanks for the review!